lib/auth0_api.rb

```diff
@@ -0,0 +1,40 @@
+require 'uri'
+require 'net/http'
+require 'auth0'
+
+class Auth0Api
+  def client
+    @client ||= new_client
+  end
+
+  def token
+    @token ||= token_data['access_token']
+  end
+
+  private
+
+  def new_client
+    Auth0Client.new(
+      client_id: ENV['AUTH0_CLIENT_ID'],
+      domain: ENV['AUTH0_DOMAIN'],
+      token: token,
+      api_version: 2,
+    )
+  end
+
+  def token_data
```

Comment:
Nice work figuring this out, @james !

Reply:
I don't understand why it's not part of the Auth0 gem!